src/instruments/src/EFB/Settings/Pages/SimOptionsPage.tsx

```diff
@@ -68,34 +68,34 @@ export const SimOptionsPage = () => {
                         </SelectGroup>
                     </SettingItem>
 
-                    <SettingItem name={t('Settings.SimOptions.EnabledMcduServerConnectionAutoDeactivatesAfter5MinutesIfNoSuccessfulConnection')}>
+                    <SettingItem name={t('Settings.SimOptions.EnableSimbridge')}>
```

Comment:
<SettingItem name={t('Settings.SimOptions.EnableSimbridge')}>
```suggestion
                    <SettingItem name={t('Settings.SimOptions.EnableSimBridge')}>
```